### PR TITLE
Add pug and style linters

### LINKED ISSUE #3544: Add Pug and Style linters (for Tikui)

### PR DESCRIPTION
**Pull request by Gnuk**
Resolves #3544

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,12 +69,18 @@
...
     "prettier-plugin-java": "1.6.2",
     "prettier-plugin-packagejson": "2.2.18",
+    "pug-lint": "^2.6.0",
     "rimraf": "3.0.2",
     "sass": "1.55.0",
     "sinon": "14.0.0",
+    "stylelint": "^14.12.1",
+    "stylelint-config-concentric-order": "^5.1.0",
+    "stylelint-config-standard": "^28.0.0",
+    "stylelint-config-standard-scss": "^5.0.0",
```
Comment (pascalgrimaud) [inline package.json:83]: prefer using fixed version

File: package.json [hunk 2]
```diff
@@ -60,12 +69,18 @@
     "jest-sonar-reporter": "2.0.0",
     "jest-transform-stub": "2.0.0",
     "lint-staged": "13.0.3",
+    "postcss-scss": "^4.0.5",
     "prettier": "2.7.1",
     "prettier-plugin-java": "1.6.2",
     "prettier-plugin-packagejson": "2.2.18",
+    "pug-lint": "^2.6.0",
```
Comment (pascalgrimaud) [inline package.json:76]: prefer using fixed version

File: package.json [hunk 3]
```diff
@@ -60,12 +69,18 @@
     "jest-sonar-reporter": "2.0.0",
     "jest-transform-stub": "2.0.0",
     "lint-staged": "13.0.3",
+    "postcss-scss": "^4.0.5",
```
Comment (pascalgrimaud) [inline package.json:72]: prefer using fixed version

**Comment by Gnuk** [comment]
I've just added pug and style linters for tikui style only at the moment because there is a lot of thing to adapt on existing scss code to have it everywhere.

But cause the style will only be in the style directory (except for the generated glyphs), I think we will no longer need to care about this.

**Comment by pascalgrimaud** [comment]
Thanks a lot for your work here @Gnuk !

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**